sympy/combinatorics/schur_number.py

```diff
@@ -51,7 +51,7 @@ def eval(cls, k):
 
     def lower_bound(self):
         f_ = self.args[0]
-        return (3**f_ - 1)/2
+        return (Integer(3)**f_ - S.One)/Integer(2)
```

Comment:
This shouldn't be needed. If `f_` is an Expr (and it should be if its in an object's `args`) the numbers combined with it will automatically be converted to SymPy types.

Reply:
It is not needed, but is it faster than relying on the automatic conversion? S.One is probably faster, for the other numbers not sure. (This is one of the cases I was talking about as not being obvious in terms of performance.)